src/dbus_consts.rs

```diff
@@ -109,9 +109,7 @@ impl HasCodes for StratisRaidType {
         match *self {
             StratisRaidType::STRATIS_RAID_TYPE_UNKNOWN => "Ok",
             StratisRaidType::STRATIS_RAID_TYPE_SINGLE => "Single",
-            StratisRaidType::STRATIS_RAID_TYPE_RAID1 => {
-                "Mirror between two disks. For 4 disks or more, they are RAID10"
-            }
+            StratisRaidType::STRATIS_RAID_TYPE_RAID1 => "Mirror between two disks",
```

Comment:
Less is more! How about just "Mirrored". Also, while you're here, shouldn't string for RAID_TYPE_UNKNOWN be something like "Unknown"?